homeassistant/components/deconz/__init__.py

```diff
@@ -44,8 +44,8 @@
 SERVICE_DATA = 'data'
 
 SERVICE_SCHEMA = vol.Schema({
-    vol.Exclusive(SERVICE_FIELD, 'deconz_id'): cv.string,
-    vol.Exclusive(SERVICE_ENTITY, 'deconz_id'): cv.entity_id,
+    vol.Optional(SERVICE_FIELD, 'deconz_id'): cv.string,
+    vol.Optional(SERVICE_ENTITY, 'deconz_id'): cv.entity_id,
     vol.Required(SERVICE_DATA): dict,
 })
```

Comment:
Use "has_at_least_one_key" to make sure that either field or entity is used.

Reply:
I was expecting comments on this. I did not know how to do it properly. I'll fix it.

Reply:
I didn't either. Just looked at alternatives to Exclusive